AD query V2-fix (#18858)
demisto/content: Packs/Active_Directory_Query/ReleaseNotes/1_4_10.md

#### Integrations
##### Active Directory Query v2

> - Added support for the *restore_user* argument in the ***enable/disable*** user command.

Reviewer: ```suggestion
- Added new argument ***restore_user*** to **ad-enable-account** and **ad-disable-account** commands
```

> - Added new argument ***restore_user*** to **ad-enable-account** and **ad-disable-account** commands.

Reviewer: ```suggestion
- Added support for ***restore_user*** to **ad-enable-account** and **ad-disable-account** commands.
```
